### DISCUSSION TITLE
GoPro Mini 11 Histogram - manually trigger or latch on?

### DISCUSSION THREAD

**Discussion by madchiller** [gopro/labs#198] [Feature Requests]
Could there be a way to  manually trigger or latch the histogram display on?  It is really difficult to set exposure in such a short window that it is displayed.. If there was some way to latch it or manually trigger it as need that would be extremely helpful.  Honestly Id love to be able to double tap the front button or something and have it latch on/off ...  but even a QR latch mode would be super useful.  Thank you!!!

**Comment by dnewman-gpsw** [comment]
Please explain "latch on".  When the histogram is on, it is there a lot, you can control the amount of time it is displayed with oMHIST=x where is fraction of system status to histogram.

**Comment by madchiller** [comment]
With the current implementation .. the histogram display is only shown for a Few seconds..  this makes setting up exposure challenging…. if we “Latch it on” …. The histogram would stay the screen until we turn off/disable the latch…..   or… maybe… as another idea..  until we hit record….     When using ND filters for example .. it’s super useful to be able to swap them quickly and pick the right ND for the exposure settings.

**Comment by dnewman-gpsw** [comment]
If I use oMHIST=1, the histogram remains on until the camera's automatic shutdown. So it is the camera's shutdown, not the histogram you are concerned about?  On Mini the shutdown is very quick, this is by design to preserve battery.  Labs has yet to alter the shutdown time.  If you used !MHIST=1, the histogram will default active, so should be straight forward to select the appropriate ND filter. Place the ND, power on, check the histogram, if need, change the ND, power on again to check.

**Reply by madchiller** [reply]
Thank you very much for your time and reply..   Hist = 1 is super helpful.. they should add this info to the main labs extensions page..  It currently only reflects : HIST=x - Displays a histogram with contrast from 1 to 11. e.g. try setting HIST to 5. HIST=0 will disable it. On 11-Mini the number is a display duration.    Any idea of when labs might be able to override the mini shut off..  I was searching and about to  make  a comment about it not working before I saw your reply.. Cheers! Thank you!
